configure.ac

```diff
@@ -5044,6 +5044,31 @@ AC_ARG_WITH(lazy_gc_thread_creation, [  --with-lazy-gc-thread-creation=yes|no
 	fi
 ], [with_lazy_gc_thread_creation=no])
 
+dnl *****************************
+dnl *** Thread suspend policy ***
+dnl *****************************
+
+dnl Set a default hybrid or cooperative suspend on some platforms
+
+dnl Coop default is set by the bitcode preset.
+
+dnl If coop isn't on by default, maybe hybrid should be?
+if test x$enable_cooperative_suspend_default != xyes; then
+	case $HOST in
+	X86 | AMD64)
```

Comment:
I'm not sure processor is relevant..but I guess ok, it disambiguates darwin, and we don't have Windows/arm support. What about Linux/arm?

Reply:
FIXME?  add host_macos, host_ios, and either remove host_darwin or make it clear host_darwin is maco | ios? And never say "osx"?

Reply:
Does coop work w/o coop handles finished? We just treat transition from jit=>native differently based on if it is a handles() transition?

Reply:
The processor check is specifically because we only want to turn this on on Intel for now - it feels like there are more deadlocks to chase down on linux on ARM.

The coop handles (plus precise managed stack scanning) would allow us to skip the "copy stackdata" step when a thread self-suspends and to stop scanning the native stack conservatively.  So not having them doesn't preclude using hybrid suspend today.